guides/common/modules/proc_configuring-rhui-alternate-content-sources.adoc

```diff
@@ -3,7 +3,7 @@
 
 .Prerequisites
 * Generate the client entitlement certificates for the required repos on the RHUA node as described in https://access.redhat.com/documentation/en-us/red_hat_update_infrastructure/4/html/configuring_and_managing_red_hat_update_infrastructure/assembly_cmg-creating-client-ent-cert-config-rpm_configuring-and-managing-red-hat-update-infrastructure#proc_cmg-creating-client-entitlement-certificate_assembly_cmg-creating-client-ent-cert-config-rpm[Creating a client entitlement certificate with the Red Hat Update Infrastructure Management Tool] in _Configuring and Managing Red Hat Update Infrastructure_.
-* Import the client entitlement certificates into your {Project}.
+* Import the client entitlement certificates into {Project}.
```

Comment:
I think that this should be a separate PR

Reply:
It's a separate commit, so I believe this is OK.